### PR TITLE
Fixed problem with apidoc

### PR DESCRIPTION
**Pull request by Dandria1802**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: home-application/pom.xml [hunk 1]
```diff
@@ -127,11 +104,23 @@
...
                                     <overWrite>true</overWrite>
-                                    <outputDirectory>${project.build.outputDirectory}/static</outputDirectory>
+                                    <fileMappers>
+                                        <org.codehaus.plexus.components.io.filemappers.FlattenFileMapper/>
+                                    </fileMappers>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.webjars.npm</groupId>
+                                    <artifactId>rapidoc</artifactId>
+                                    <version>7.2.1</version>
```
Comment (Romanov-Niko) [inline home-application/pom.xml]: Versions should be in a separate section

File: home-application/pom.xml [hunk 2]
```diff
@@ -111,7 +88,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-                <version>2.8</version>
+                <version>3.1.2</version>
```
Comment (Romanov-Niko) [inline home-application/pom.xml]: Versions should be in a separate section

**Review by Romanov-Niko — CHANGES_REQUESTED** [review]

**Review by kh0ma — APPROVED** [review]

**Review by Abhai2016 — APPROVED** [review]

File: home-application/pom.xml (hunk 1 above)
Comment (Dandria1802) [reply home-application/pom.xml]: Done.

File: home-application/pom.xml (hunk 2 above)
Comment (Dandria1802) [reply home-application/pom.xml]: Done.

**Review by Romanov-Niko — APPROVED** [review]

**Review by OlTym — APPROVED** [review]

**Review by PoLytvy — APPROVED** [review]

**Pull request merged**